packages/veto-backend/src/api/cyphernode/client.ts

```diff
@@ -19,8 +19,8 @@ const httpsAgent = new https.Agent({
  * Cyphernode has a weird implementation of JWT at the moment.
  * TODO: Implement standard JWT signing when cyphernode updates this.
  */
-function getBearerToken(key: string, keyId: string): string {
-  const current = Math.round(new Date().getTime() / 1000) + 10
+export function getBearerToken(key: string, keyId: string): string {
```

Comment:
I don't see it used anywhere else, should it be exported?

Reply:
nope my bad